client/src/app/shared/shared-share-modal/video-share.component.ts

```diff
@@ -204,15 +206,19 @@ export class VideoShareComponent {
     if (this.playlist) {
       this.playlistUrl = await this.getPlaylistUrl()
       this.playlistEmbedUrl = await this.getPlaylistEmbedUrl()
-      this.playlistEmbedHTML = await this.getPlaylistEmbedCode()
-      this.playlistEmbedSafeHTML = this.sanitizer.bypassSecurityTrustHtml(this.playlistEmbedHTML)
+      this.playlistEmbedHTML = await this.getPlaylistEmbedCode(this.customizations.responsive)
+      this.videoEmbedSafeHTML = this.sanitizer.bypassSecurityTrustHtml(this.customizations.responsive
```

Comment:
Typo with `playlistEmbedSafeHTML`? And why do you need to call again `getPlaylistEmbedCode`?

Reply:
Typo fixed in https://github.com/Chocobozzz/PeerTube/pull/5690/commits/c562b73240fd9010120520411e0e68a1d9a8d6eb. `playlistEmbedSafeHTML` is used to preview the embed in the share modal. I did not succeed to display the responsive preview correctly. So if you check the 'responsive embed' button, it will bring the correct responsive embed code, but the preview will remain non-responsive.

Reply:
Done in https://github.com/Chocobozzz/PeerTube/pull/5690/commits/c26c1a5f735bd6ca1d4957072a741b1f8c953e23